### PR TITLE
Tesla Grenade + Cavern Dwellers

### PR DESCRIPTION
**Pull request by Geevies**
* Added tesla grenades.
* Cavern Dwellers now throw a bio version of the tesla grenade.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm [hunk 1]
```diff
@@ -47,9 +50,17 @@
 	see_in_dark = 8
 	see_invisible = SEE_INVISIBLE_NOLIGHTING
 
+	butchering_products = list(/obj/item/grenade/tesla/bio = 1)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm:53]: I think getting a grenade for every mob butchered is a bit too much considering how many dwellers there may be. Also how tf do you get an entire grenade from a dweller?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — COMMENTED** [review]
I still think 60% is way too high. Should be like 30 at most.

**Review by JohnWildkins — COMMENTED** [review]
Seems to work but the grenades are _extremely_ powerful (if somewhat RNG because, tesla). I punched a cavern dweller on runtime and it proceeded to lob tesla grenades at me like a Mark 19 while I sat in perma-crit from the first one nearly instakilling me. I don't like the implication of them being a non-dweller item that could eventually find itself being acquired in any fashion other than by adminbus, because in this state they feel giga-powerful

**Pull request closed without merging**